### ISSUE TITLE
[Question] How did you know the url string for player valorant store?

### ISSUE THREAD

**Issue by gupta-shrinath** [TewPingz/ValorantAPI#4]
What tools were used?I would like to learn how can we extract the private apis and understand the api authentication

**Comment by Koupah** [comment]
> What tools were used?I would like to learn how can we extract the private apis and understand the api authentication

You just have to sniff the socket or whatever it is that Valorant uses to see them. You're honestly better off just looking at other peoples repos and/or asking other developers.

**Comment by gupta-shrinath** [comment]
Yeah I get that. I would love to know about monitoring the network traffic of an application. This seems to be a great place to learn from.
